plugins/modules/apt_rpm.py

```diff
@@ -164,16 +203,32 @@ def main():
 
     p = module.params
 
+    out = ""
     if p['update_cache']:
-        update_package_db(module)
+        out += update_package_db(module)
 
+    if p['clean']:
+        out += clean(module)
+
+    if p['dist_upgrade']:
+        out += dist_upgrade(module)
+
+    # If package is missing return exit_json
     packages = p['package']
 
-    if p['state'] in ['installed', 'present']:
-        install_packages(module, packages)
+    if not isinstance(packages, list):
+        if out == '':
+            module.fail_json(msg="Missing command")
```

Comment:
Why fail? Is it necessary that something is done? Why not simply call `module.exit_json` with `changed=False`?